### PR TITLE
CP for #300: Enable recall/latency charts on results

### PR DESCRIPTION
**Pull request by filipecosta90**
This PR cherry-picks the proposed changes in https://github.com/erikbern/ann-benchmarks/pull/300 into multiclient_tool

### CODE REVIEW

File: ann_benchmarks/plotting/utils.py [hunk 1]
```diff
@@ -45,6 +45,8 @@ def compute_metrics(true_nn_distances, res, metric_1, metric_2,
         algo_name = properties['name']
         # cache distances to avoid access to hdf5 file
         run_distances = numpy.array(run['distances'])
+        # cache times to avoid access to hdf5 file
+        times = numpy.array(run['times'])
         if recompute and 'metrics' in run:
             del run['metrics']
         metrics_cache = get_or_create_metrics(run)
```
Comment (GuyAv46) [inline ann_benchmarks/plotting/utils.py:52]: missing `times` property on lines 54-59

**Review by GuyAv46 — COMMENTED** [review]
Looks good. One fix

File: ann_benchmarks/plotting/utils.py (hunk 1 above)
Comment (filipecosta90) [reply ann_benchmarks/plotting/utils.py:52]: fixed on https://github.com/RedisAI/ann-benchmarks/pull/26/commits/cf559609a64d9ac633dfda2a33a40a098ab90bb9

**Review by GuyAv46 — APPROVED** [review]
LGTM

**Pull request merged**